pkg/obfuscator/mac_address_test.go

```diff
@@ -34,17 +42,72 @@ func TestMacReplacementSuper(t *testing.T) {
 	}{
 		{name: "squashed", input: "69806FE67C05", expectedOutput: "69806FE67C05"},
 		{name: "squashed-lowercase", input: "69806fe67c05", expectedOutput: "69806fe67c05"},
-		{name: "uppercase-colon", input: "69:80:6F:E6:7C:05", expectedOutput: StaticMacReplacement},
-		{name: "lowercase-dash", input: "eb-a1-2a-b2-09-bf", expectedOutput: StaticMacReplacement},
-		{name: "lowercase-colon", input: "eb:a1:2a:b2:09:bf", expectedOutput: StaticMacReplacement},
-		{name: "multi-colon", input: "eb:a1:2a:b2:09:bf eb:a1:2a:b2:09:bf", expectedOutput: StaticMacReplacement + " " + StaticMacReplacement},
-		{name: "multi-colon-dash", input: "16-7C-44-26-24-14 BF:51:A4:1B:7D:0B", expectedOutput: StaticMacReplacement + " " + StaticMacReplacement},
-		{name: "mac surrounded", input: "mac 52:df:20:08:6c:ff caused some trouble", expectedOutput: fmt.Sprintf("mac %s caused some trouble", StaticMacReplacement)},
+		{name: "uppercase-colon", input: "69:80:6F:E6:7C:05", expectedOutput: staticMacReplacement},
+		{name: "lowercase-dash", input: "eb-a1-2a-b2-09-bf", expectedOutput: staticMacReplacement},
+		{name: "lowercase-colon", input: "eb:a1:2a:b2:09:bf", expectedOutput: staticMacReplacement},
+		{name: "multi-colon", input: "eb:a1:2a:b2:09:bf eb:a1:2a:b2:09:bf", expectedOutput: staticMacReplacement + " " + staticMacReplacement},
+		{name: "multi-colon-dash", input: "16-7C-44-26-24-14 BF:51:A4:1B:7D:0B", expectedOutput: staticMacReplacement + " " + staticMacReplacement},
+		{name: "mac surrounded", input: "mac 52:df:20:08:6c:ff caused some trouble", expectedOutput: fmt.Sprintf("mac %s caused some trouble", staticMacReplacement)},
 		{name: "mac as guid", input: "4a5299ac-6104-479d-aed4-b79faedffcb4", expectedOutput: "4a5299ac-6104-479d-aed4-b79faedffcb4"},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
-			o := NewMacAddressObfuscator()
+			o, _ := NewMacAddressObfuscator(schema.ObfuscateReplacementTypeStatic)
 			assert.Equal(t, tc.expectedOutput, o.Contents(tc.input))
 		})
 	}
 }
+
+func TestMACConsistentObfuscator(t *testing.T) {
```

Comment:
Add a test for the same mac address with dashes and colons.

Reply:
Done